fedoracommunity/search/index.py

```diff
@@ -60,18 +62,22 @@ def download_file(url, dest):
 class Indexer(object):
     def __init__(self, cache_path,
                  tagger_url=None,
-                 pkgdb_url=None,
+                 bodhi_url=None,
                  mdapi_url=None,
-                 icons_url=None):
+                 icons_url=None,
+                 pdc_url=None,
+                 pagure_url=None):
 
         self.cache_path = cache_path
         self.dbpath = join(cache_path, 'search')
         self.icons_path = join(cache_path, 'icons')
         self.default_icon = 'package_128x128.png'
         self.tagger_url = tagger_url or "https://apps.fedoraproject.org/tagger"
-        self.pkgdb_url = pkgdb_url or "https://admin.fedoraproject.org/pkgdb"
+        self.bodhi_url = bodhi_url or "https://bodhi.fedoraproject.org"
         self.mdapi_url = mdapi_url or "https://apps.fedoraproject.org/mdapi"
         self.icons_url = icons_url or "https://alt.fedoraproject.org/pub/alt/screenshots"
+        self.pdc_url = pdc_url or "https://pdc.fedoraproject.org/rest_api/v1"
+        self.pagure_url = pagure_url or "http://pkgs.stg.fedoraproject.org/pagure/api/0"
```

Comment:
Yeah, might as well.

@yashvardhannanavati, can you switch this to ``https://pkgs.fedoraproject.org....`` to use prod instead of staging.

Prod isn't up yet, but this is a good default for the future.